apps/src/templates/SchoolInfoConfirmationDialog.jsx

```diff
@@ -21,24 +21,45 @@ class SchoolInfoConfirmationDialog extends Component {
 
   constructor(props) {
     super(props);
-
     this.state = {
       showSchoolInterstitial: false,
       schoolName: props.schoolName,
       isOpen: true
     };
   }
 
+  static defaultProps = {
```

Comment:
How do these defaultProps get used?

Reply:
@epeach The default props will be used unless the parent component over-rides them.